classes/decision/types/BackToCopyediting.inc.php

```diff
@@ -147,6 +148,14 @@ public function getSteps(Submission $submission, Context $context, User $editor,
         return $steps;
     }
 
+    /**
+     * Determine if can back out to copy editing
+     */
+    public function canRetract(Submission $submission, ?int $reviewRoundId): bool
+    {
+        return true;
+    }
```

Comment:
This is needed so that we can determined which retractable/backable decision we can have for a given stage where the submission is right now . otherwise we have to do a lot of if/else type code checking this makes the code a bit messy . which this interface/contract implementation, we can easily determine it dynamically.

Reply:
As far as I can see, this is the only place that the interface is used:

https://github.com/touhidurabir/pkp-lib/blob/6e6f6bbce9697b098021de113998372355ce6fd1/pages/decision/DecisionHandler.inc.php#L107-L110

Wouldn't removing the `DecisionRetractable` interface lead to the same outcome? In other words, `canRetract()` doesn't need to be run for this decision.